src/modules/esl-event-listener/README.md

```diff
@@ -1,50 +1,211 @@
+  a target relatively to `host | host.$host` object or absolutely in bounds of the active DOM tree.
+  The `target` key is also supports an exact reference(s) for `EventTargets`. 
+
+  ⚠ Any `EventTarget` or event ESL `SynteticEventTarget` (including [`ESLMediaQuery`](../esl-media-query/README.md)) 
+  can be a target for listener API.
+  
+  The `target` property can be declared via `PropertyProvider` as well.  
+  
+- #### `selector` key
+  <u>Type:</u> `string | PropertyProvider<string>`  
+  <u>Default Value:</u> `null`  
+  <u>Description:</u> the CSS selector to check the event bubbling chain.
+  The property to use OOTB event delegation check of target.
+  
+  Supports `PropertyProvider` to declare the computed value as well.
+  
+- #### `capture` key
+  <u>Type:</u> `boolean`  
+  <u>Default Value:</u> `false`  
+  <u>Description:</u> marker to use capturing phase of the DOM event to handle.
+ 
+- #### `passive` key
+  <u>Type:</u> `boolean`  
+  <u>Default Value:</u> `false`  
+  <u>Description:</u> marker to use passive subscription to the native event.
+
+  ⚠ ESL uses passive subscription by default for `wheel`, `mousewheel`, `touchstart`, `touchmove` events.
+  You need to declare `passive` key explicitly to override such behavior.
+  
+- #### `once` key
+  <u>Type:</u> `boolean`  
+  <u>Default Value:</u> `false`  
+  <u>Description:</u> marker to unsubscribe listener with the first successful handling of the event.
+
+- #### `auto` key (for `ESLEventDesriptorFn` declaration only)
+  <u>Type:</u> `boolean`  
+  <u>Default Value:</u> `false` for `ESLEventUtils.initDescriptor`, `true` for `@listen`  
+  <u>Description:</u> marker to make an auto-subscribable descriptor. See [Automatic (collectible) descriptors](#automatic-collectible-descriptors).
+  
+- #### `inherit` key (for `ESLEventDesriptorExt` only)
+  <u>Type:</u> `boolean`  
+  <u>Description:</u> available in extended version of `ESLEventDesriptor` that used in descriptor declaration API.
+  Allows to inherit `ESLEventDesriptor` data from the `ESLEventDesriptorFn` declared with the same key in the property chain 
+  of currently decorated  `ESLEventDesriptorFn` holder. See `initDescriptor` usages example.
+  
+
+
+### <a name="automatic-collectible-descriptors">Automatic (collectible) descriptors</a>
+
+If the `ESLEventDesriptorFn` is declared with the `auto` marker of the associated definition 
+it became auto-collectable for the ESL event listeners module.
+Auto-collectable (or auto-subscribable) descriptors should be declared with `ESLEventUtils.initDescriptor` or
+`@listen` decorator as they are also stored under the special private collection of the holder object.
+That means you can make all auto-collectable descriptors to be subscribed at once in the future.
+The `ESLBaseElment` and `ESLMixinElement` do it in the `connectedCallback` OOTB.
+See the usage of [`ESLEventUtils.subscibe`](#-esleventutilssubscribe) for more details.
```

Comment:
`(#-esleventutilssubscribe)` - does it work?

Reply:
Not at all )
Create #1457 and temporarily made a workaround on the page.